source/Hardpoint.cpp

```diff
@@ -218,16 +218,18 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	// offset of (.5 * velocity) and that velocity includes the velocity of the
 	// ship that fired them.
```

Comment:
GitHub won't let me push a suggestion high enough, but this comment should now be removed since I've suggested it be moved down.